src/pages/Product/Product.scss

```diff
@@ -0,0 +1,42 @@
+.Product {
+  &__container {
+    width: 80%;
+    margin: 0 auto;
+  }
+
+  &__name {
+    font-size: 0.9375rem;
+    color: $gray-dim;
+    font-weight: bolder;
+    line-height: normal;
+    margin-top: 1.3125rem;
+    margin-bottom: 1.4375rem;
```

Comment:
These values are very specific, can they be rounded?

Reply:
@Israel-Laguan By the moment I haven't found a way to convert JSON to an SCSS file, I could search it again, but I think an SCSS alternative would be to create a file for specific variables and create there the [SCSS variables](https://sass-lang.com/documentation/variables) , what you think?

For example, could create a "sizes.scss"

```scss
// sizes
$font-small: 0.875rem;
$font-normal: 1rem;
$font-xl: 1.25rem;
```

And then just use it on my files

```scss
font-size: $font-xl;
```

Reply:
I think we can do the refactor for size, color, and other values later, maybe handled on this issue https://github.com/Azordev/did-app/issues/33

Reply:
I've rounded the rem values